advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: 'Using Commvault Express'
+description: 'Walkthroughs of multiple Commvault Express usage scenarios'
+---
+
+This section shows how backups are taken and restored from EDB Postgres Advanced Server (EPAS) using Commvault Express.
```

Comment:
How to backup and restore from an EDB Postgres Advanced Server using Commvault Express.

don't need "this sections shows"